README.md

```diff
@@ -120,8 +121,19 @@ If you find an issue with this package and have a fix, please feel free to open
 
 ## Testing
 
-If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`. 
-* Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.
+
+- Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+
+### Debug Logs
+
+To display the debug logs for this library, set the `DD_CONSTRUCT_DEBUG_LOGS` env var to `true` when running `cdk synth`. 
```

Comment:
I dont know how important this is but I tried the Debugs logs myself and felt that the Debug logs are "overpowered" by the cdk output like the CloudFormation template. I dont think there is any way to get around that but maybe adding a note that the debug logs appear before the CloudFormation template might be helpful. 

People who enable the debug logs might run the synth command but be overwhelmed with cloudformation template and not even know to look above that to read the debug logs. I don't know its just something that was on my mind.

Reply:
Good catch, let's also mention a way to suppress the template output. This won't normally be required because you're mostly concerned with debug logs when running into issues (in which case you won't see the template output).